app/decorators/user_decorator.rb

```diff
@@ -0,0 +1,17 @@
+class UserDecorator < Draper::Decorator
+  decorates_association :destinations
+  delegate_all
+  def list_last_ten
+    if last_ten_destinations.empty?
+      h.content_tag(:h4, "You have no trips")
+    else
+      last_ten_destinations.map do |d|
+        h.content_tag(:div, "Trip Name: #{d.name}", class: 'destination').html_safe
+      end.join.html_safe
+    end
+  end
+
```

Comment:
I like this setup and logic! I too wonder about the html_safe - just want to know it's use.